README.md

```diff
@@ -108,7 +108,7 @@ later.
 
 ## Installing NEST
 
-Please see the online [NEST Installation Instructions](http://www.nest-simulator.org/installation)
+Please see the online [NEST Installation Instructions](https://nest-simulator.readthedocs.io/en/stable/installation)
```

Comment:
```suggestion
Please see the online [NEST Installation Instructions](https://nest-simulator.org/documentation/installation)
```
We want to avoid pointing into the readthedocs hierarchy, since this might change outside our control. The Installation link is broken because it doesn't point into the documentation and was formerly a direct proxy redirect.